LonaStudio/Preferences/CSColors.swift

```diff
@@ -25,7 +25,7 @@ class CSColors: CSPreferencesFile {
         guard let colorData = data["colors"] else { return [] }
         
         return colorData.arrayValue.map({ color in
-            let id = color["id"]?.string
+            let id = color["id"]!.string!
```

Comment:
We probably shouldn't force unwrap here, since if the user has a misconfigured `cscolors.json`, they'll get a crash.